Added chapter on Data curation (#2817)
alan-turing-institute/the-turing-way: book/website/reproducible-research/rdm/rdm-data-curation.md

(rr-rdm-data-curation)=
# Data Curation

> Data curation is one of the key aspects of RDM. It  involves capturing of research data, appraisal, disposal, description, preservation, access, reuse and transformation. Data curation promotes the use of data from the point of creation to ensuring that its used for the purpose it is intended for.  Data curation  enables you to easily  access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. You can imagine if you don't follow the data curation process, it becomes difficult to access that data.

Reviewer: ```suggestion
Data curation is one of the key aspects of Research Data Management (RDM). It involves the capturing, appraisal, disposal, description, preservation, access, reuse and transformation of research data. Data curation promotes the use of data from the point of creation to ensuring that its used for the purpose it is intended for. Data curation also  enables you to easily access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. You can imagine that if you don't follow the data curation process, it becomes difficult to access that data.
```

> Data curation is one of the key aspects of RDM. It  involves capturing of research data, appraisal, disposal, description, preservation, access, reuse and transformation. Data curation promotes the use of data from the point of creation to ensuring that its used for the purpose it is intended for.  Data curation  enables you to easily  access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. You can imagine if you don't follow the data curation process, it becomes difficult to access that data.

Reviewer 2: Please spell out RDM on the first line, then RDM in brackets.

Reviewer 3: ```suggestion
Data curation is one of the key aspects of Research Data Management (RDM). It  involves capturing of research data, appraisal, disposal, description, preservation, access, reuse and transformation. Data curation promotes the use of data from the point of creation to ensuring that its used for the purpose it is intended for.  Data curation  enables you to easily  access data sets and information that you need in an organised format, you can easily extract data from whichever repository it is stored. You can imagine if you don't follow the data curation process, it becomes difficult to access that data.
```

Suggesting the change mentioned by Emma. You can commit changes if this is right.

Reviewer 3: Resolved by Jen's edits.

> ### 1. Research data capture

Reviewer: ```suggestion
## Research Data Management Pipeline

### 1. Data capture
```

> Data capture is a process of gathering or collecting data from different sources which will be processed and used for certain purposes. These sources may be research articles in electronic format; databases like electronic databases among others. This stage focuses on ensuring that the data captured is actually fit for the purpose and ready for curation.

Reviewer 2: ```suggestion
Data capture is the process of gathering or collecting data from different sources, which will be processed and used for certain purposes. These sources may be research articles in electronic format or databases like electronic databases among others. This stage focuses on ensuring that the data captured is actually fit for purpose and ready for curation.
```

### 2. Data appraisal

> Data appraisal and selection are fundamental stages in the data curation process. In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing; checking, validating and cleaning the data; in addition to anonymizing  the data where necessary; describing the data; and managing  and storing the data.  As you appraise the data you need to adhere to the documented guidance, policies and legal requirements in evaluating research data in a particular organization and selecting the data that requires long term preservation. Your  Appraisal and selection policy should ensure consistency, transparency and accountable decision making.

Reviewer: ```suggestion
Data appraisal and selection are fundamental stages in the data curation process. In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing; checking, validating and cleaning the data; in addition to anonymizing  the data where necessary; describing the data; and managing  and storing the data.  As you appraise the data you need to adhere to the documented guidance, policies and legal requirements in evaluating research data in a particular organization and selecting the data that requires long term preservation. Your appraisal and selection policy should ensure consistency, transparency, and accountable decision making.
```

### 3. Data disposal 

> Data disposal involves disposing of data that has not be selected for retention. You need to have an appraisal policy which will guide you on the data required for archiving, redeployment, transfer of custody or ownership or destroying the research Data 

Reviewer: ```suggestion
Data disposal involves disposing of data that has not be selected for retention. You need to have an appraisal policy which will guide you on the data required for archiving, redeployment, transfer of custody or ownership or destroying the research data.
```

> Data disposal involves disposing of data that has not be selected for retention. You need to have an appraisal policy which will guide you on the data required for archiving, redeployment, transfer of custody or ownership or destroying the research Data 

Reviewer 2: ```suggestion
Data disposal involves disposing of data that has not be selected for retention. You need to have an appraisal policy, which will guide you on the data required for archiving, redeployment, transfer of custody or ownership or destroying the research data. 
```

### 4. Data description

> Data description requires that you are able to interpret the data; derive data; produce research outputs; author publications; data anonymisation; data visualization; data validation  and prepare the data for preservation. So  its important  you describe the research data so that it is discoverable and usable over time. There are also  metadata standard that already exist to help you with standardised descriptions. 

Reviewer: ```suggestion
Data description requires that you are able to interpret the data; derive data; produce research outputs; author publications; data anonymisation; data visualization; data validation  and prepare the data for preservation. So it is important that you describe the research data so that it is discoverable and usable over time. There are also  metadata standard that already exist to help you with standardised descriptions. 
```

> Data description requires that you are able to interpret the data; derive data; produce research outputs; author publications; data anonymisation; data visualization; data validation  and prepare the data for preservation. So  its important  you describe the research data so that it is discoverable and usable over time. There are also  metadata standard that already exist to help you with standardised descriptions. 

Reviewer 2: ```suggestion
Data description requires that you are able to interpret the data, derive data, produce research outputs, author publications, data anonymisation, data visualization, data validation and prepare the data for preservation. Therefore, it is important that you describe the research data thoroughly so that it is discoverable and usable over time. There are also metadata standards that already exist to help you with standardised descriptions of the dataset. 
```

### 5. Data preservation

> Please find more information using this link:

Reviewer: ```suggestion
Please find more information in the chapter on [Data Management Plans](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-dmp.html).
```

> Please find more information using this link:

Reviewer 2: ```suggestion
Please find more information on data preservation using this link:
```

> https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-dmp.html

Reviewer: ```suggestion
```

### 6. Data access 

> Data access entails distributing data, sharing data, publishing data, linking data to outputs, controlling access, establishing copyright and promoting or disseminating the data to wider audiences to access it or re-use it.  You can make the  data freely available online  to anyone  who may be interested in reading it or you may restrict access of the data or provide an option of the data accessible.

Reviewer: ```suggestion
Data access entails distributing data, sharing data, publishing data, linking data to outputs, controlling access, establishing copyright and promoting or disseminating the data to wider audiences to access it or re-use it.  You can make the data freely available online to anyone who may be interested in reading it or you may restrict access of the data or provide an option of the data accessible.
```

> Data access entails distributing data, sharing data, publishing data, linking data to outputs, controlling access, establishing copyright and promoting or disseminating the data to wider audiences to access it or re-use it.  You can make the  data freely available online  to anyone  who may be interested in reading it or you may restrict access of the data or provide an option of the data accessible.

Reviewer 2: ```suggestion
Data access entails distributing data, sharing data, publishing data, linking data to outputs, controlling access, establishing copyright and promoting or disseminating the data to wider audiences to access it or re-use it.  You can make the data freely available online to anyone who may be interested in reading it or you may restrict access of the data or provide an option of how to access the data.
```

> Read more information here:

Reviewer 2: add link here

### 7. Data transformation

> Data transformation is a practice of examining large datasets to generate new information. You can re-analysis the research data and make relationships that may not have been previously discovered. 

Reviewer: ```suggestion
Data transformation is a practice of examining large datasets to generate new information. You can reanalyze the research data and make relationships that may not have been previously discovered. 
```

> Data transformation is a practice of examining large datasets to generate new information. You can re-analysis the research data and make relationships that may not have been previously discovered. 

Reviewer 2: ```suggestion
Data transformation is the practice of examining large datasets to generate new information. You can re-analysis the research data and make relationships that may not have been previously discovered. 
```

You can imagine that if you don't follow the data curation process, it becomes difficult to access that data.

## Data Curation Pipeline

### 1. Data capture
Data capture is the process of gathering or collecting data from different sources, which will be processed and used for certain purposes. 
These sources may be research articles in electronic format or databases like electronic databases among others. 
This stage focuses on ensuring that the data captured is actually fit for purpose and ready for curation.

### 2. Data appraisal
Data appraisal and selection are fundamental stages in the data curation process. 
In data appraisal, you are required to select the appropriate data by entering, digitizing, transcribing, checking, validating and cleaning the data; in addition to anonymizing the data where necessary, describing the data and managing and storing the data.  
As you appraise the data you need to adhere to the documented guidance, policies and legal requirements in evaluating research data in a particular organization and selecting the data that requires long term preservation. 

> Your appraisal and selection policy should ensure consistency, transparency and accountable decision making.

Reviewer 4: ```suggestion
Your appraisal and selection policy should ensure consistency, transparency, and accountable decision making.
```

### 3. Data disposal 
Data disposal involves disposing of data that has not be selected for retention. 
You need to have an appraisal policy which will guide you on the data required for archiving, redeployment, transfer of custody or ownership or destroying the research data.

### 4. Data description
Data description requires that you are able to interpret the data; derive data; produce research outputs; author publications; data anonymisation; data visualization; data validation and prepare the data for preservation. 
So it is important that you describe the research data so that it is discoverable and usable over time. 
There are also  metadata standards that already exist to help you with standardised descriptions. 

### 5. Data preservation

> Please find more information in the chapter on [Data Management Plans](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-dmp.html).

Reviewer 4: Using the syntax here for linking across the book: https://the-turing-way.netlify.app/community-handbook/style/style-crossref.html
```suggestion
Please find more information in the chapter on {ref}`Data Management Plans<rr-rdm-dmp>`.
```
